eq-author-api/migrations/addExpressionOperator.js

```diff
@@ -0,0 +1,16 @@
+//This is an auto-generated file.  Do NOT modify the method signature.
+
+module.exports = function addExpressionOperator(questionnaire) {
+  questionnaire.sections.map(section => {
+    section.pages.map(page => {
+      if (page.routing) {
+        page.routing.rules.map(rule => {
+          if (!rule.expressionGroup.operator) {
+            rule.expressionGroup.operator = "And";
```

Comment:
Should this refer to the AND constant?

Reply:
There is only a constant for it on the publisher side atm, but I will add one for API and frontend.

Reply:
I have added the constant now.